src/drawers/elementDrawer.ts

```diff
@@ -60,6 +60,15 @@ export module _Drawer {
     public _getSelector(): string {
       return this._svgElement;
     }
+
+    public _getPixelPoint(datum: any, index: number): Point {
+      switch(this._svgElement) {
+        case "circle":
```

Comment:
I realize that, but it's still wrong. Make another PR to address this specific issue?